descholar-ceo/private-events
# Add private eventbrite logic

## What does this PR do?
It adds the entire `private event-brite` logic

## Tasks to be completed in this PR
- [x] Project setup
- [x] Write the actual logic of the private Eventbrite
- [x] Add some stylings
- [x] Fix some linters error
- [x] Polish codes

## How to test this PR on your local machine?
1. Clone this repository with `git clone https://github.com/descholar-ceo/private-events.git`
1. Go inside this repo locally by typing `cd private-events`
1. Check out this branch by running `git checkout add-events-brite-logic`
1. Run `bundle install` to install all ruby gems
1. Run `yarn` to install Javascript dependencies
1. Run `bin/rails db:migrate` to create necessary tables in the database
1. Run `rails server`
1. Navigate to `http://127.0.0.1:3000` in your browser
1. Signup and create some events
1. To create an invitation click the `invite someone` in the navigation bar
1. To see a list of all events, click on `All events` button on the navbar
1. To see event's show page, navigate to the `all events` page and click on any event's `show` button
